src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedService.java

```diff
@@ -98,4 +115,37 @@ private void logUnlinkedCaseEventAgainstOriginalCaseIfQidIsNowLinkedToNewCase(
           messageTimestamp);
     }
   }
+
+  private boolean checkRequiredFieldsForIndividualHI(
+      String questionnaireId, String caseType, String individualCaseId) {
+    if (isIndividualQuestionnaireType(questionnaireId)
+        && caseType.equals("HH")
+        && individualCaseId != null) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  private boolean checkIfIndQIDAndIndCaseIdPresentButCaseTypeNotHH(
+      String questionnaireId, String caseType, String individualCaseId) {
+    if (isIndividualQuestionnaireType(questionnaireId)
+        && !caseType.equals("HH")
+        && individualCaseId != null) {
+      return true;
+    } else {
+      return false;
```

Comment:
Same as @LukeLoze1 suggested above, you can return the conditional instead of `if ... return true`